packages/resource-pooling/resource-pooling.1.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Jan Rochel <jan@besport.com>"
+authors: [ "Jan Rochel (BeSport)" ]
+synopsis: "Library for pooling resources like connections, threads, or similar"
+description: "This package is derived from the module Lwt_pool from the lwt package, which implements resource pooling. With Resource_pool this package provides a modified version with additional features. Also there is a module called Server_pool that manages resource clusters, specifically a cluster of servers each with its own connection pool."
+license: "MIT"
+homepage: "https://github.com/ocsigen/resource-pooling"
+dev-repo: "git+https://github.com/ocsigen/resource-pooling.git"
+bug-reports: "https://github.com/ocsigen/resource-pooling/issues"
+build: [["dune" "build" "-j" jobs]]
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune"
```

Comment:
As a side note, for the next release you should think about putting the `dune-project` file into the repository as it is required for dune 2.0 and also helps check (for us) which minimal version of dune is needed.

Reply:
Many thanks! All suggestions have been applied.

Reply:
Done. Sorry for all the trouble.

Reply:
No worries